mypy.ini

```diff
@@ -13,7 +13,6 @@ strict_equality=True
 
 [mypy-tests.*]
 disallow_untyped_calls=True
-disallow_incomplete_defs=True
```

Comment:
This was forcing me to add `-> None` to every pytest method, which seemed silly.